cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -79,8 +84,6 @@
   FREQUENCY("frequency"),
 
   NAMESPACE_NAME("namespace-name"),
-  NAMESPACE_DESCRIPTION("description"),
-  NAMESPACE_PRINCIPAL("principal"),
```

Comment:
I just refactored and renamed it and moved in the same file in different section. So they are not actually removed.